src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/resource_allocation_manager.h

```diff
@@ -160,9 +160,14 @@ class ResourceAllocationManager {
   /**
    * @brief Create and send OnRCStatusNotification to mobile and HMI
    * @param event trigger for notification sending
+   * @param application - application to which notification is sent
```

Comment:
could you please rephrase this, I'm not quite understand

Reply:
@yang1070 please look at 63375e52c51679b0bbf44e9748a9bfff25a3564e